Cargo.toml

```diff
@@ -138,6 +138,7 @@ tedge_config_macros = { path = "crates/common/tedge_config_macros" }
 tedge_config_macros-impl = { path = "crates/common/tedge_config_macros/impl" }
 tedge_config_manager = { path = "crates/extensions/tedge_config_manager" }
 tedge_downloader_ext = { path = "crates/extensions/tedge_downloader_ext" }
+tedge-configuration-plugin = { path = "plugins/tedge_configuration_plugin" }
```

Comment:
A bit surprised that 'cargo sort' didn't complain here

Reply:
There are actually a few dependencies which are not strictly sorted alphabetically due to differences in dashes and underscores:

![image](https://github.com/thin-edge/thin-edge.io/assets/3029781/f266bd38-b6c7-4f8e-b620-c4500ba5af4a)

I'm tempted to leave it as is since for now.

Reply:
Reading the docs on [cargo-sort](https://crates.io/crates/cargo-sort), it looks like the "workspace.dependencies" property is not monitored by cargo-sort...hence why we have some imports out of order.